src/utilities/divide/divide.test.js

```diff
@@ -0,0 +1,40 @@
+import { divide } from './divide';
+
+describe('The divide function', () => {
+  describe('when the numbers do not equal zero', () => {
+    it('should return a valid result', () => {
+      const result = divide(10, 5);
+      expect(result).toBe(2);
+    });
+  });
+
+  describe('when the first number equals zero', () => {
+    describe('when the second number does not equal zero', () => {
```

Comment:
Here, you say:
```
The divide function
when the first number equals zero
when the second number does not equal zero
```

I think it would sound better with the word "and"

```
The divide function
when the first number equals zero
and when the second number does not equal zero
```